Tests/Tests/CustomAttributeSpec.swift

```diff
@@ -43,6 +43,14 @@ class CustomAttributeSpec: QuickSpec {
                 }
             }
 
+            context("when accessing dictionary representation") {
+
+                it("should return dictionary without type") {
```

Comment:
Can you add one more test that checks the expected value of `dictionaryRepresentation`?

Reply:
Test added [here](https://github.com/rakutentech/ios-inappmessaging/pull/253/commits/bba9dcc57d5ffb7fc5416a53bdff7b5809163f5c).